cltkreaders/lat.py

```diff
@@ -16,12 +17,12 @@
 from cltk.dependency.processes import LatinStanzaProcess
 from cltk.sentence.lat import LatinPunktSentenceTokenizer
 from cltk.tokenizers.lat.lat import LatinWordTokenizer
-
 from cltk.utils import get_cltk_data_dir, query_yes_no
 from cltk.data.fetch import FetchCorpus
 from cltk.core.exceptions import CLTKException
 
-class LatinTesseraeCorpusReader(TesseraeCorpusReader):
+
+class LatinTesseraeCorpusReader(TesseraeCorpusReader, ABC):
```

Comment:
PyCharm told me to add this ABC, I am not sure why. You're the OOP guy, so I defer to you. But if not this, should there be a `super()`?